HealthAngels.SignedSessions/Session/CustomRedisCache.cs

```diff
@@ -13,6 +13,8 @@ public class CustomRedisCache : IDistributedCache
         private readonly IDistributedCache _cache;
         private readonly ISignatureHelper _signatureHelper;
 
+        public CustomRedisCache()
```

Comment:
Please rename the class + file to CustomDistributedCache (it's a wrapper around IDistributedCache, not particular around RedisCache)

Reply:
As discussed I have created a ICustomRedisCache which implements IDistributedCache.